pkgs/applications/video/mpv/scripts/memo.nix

```
@@ -0,0 +1,34 @@
+{ fetchFromGitHub
+, buildLua
+, lib
+, unstableGitUpdater
+}:
+
+buildLua {
+  pname = "memo";
+  version = "unstable-2023-10-31";
+
+  src = fetchFromGitHub {
+    owner = "po5";
+    repo = "memo";
+    rev = "6f2b036ef860e130ea584657389e0a7ceb4d954f";
+    sha256 = "03rk1f97jblnajgwr36qz7hq1qlxrgkywc0gnq0wwg1vxiga9j4v";
+  };
+  passthru.updateScript = unstableGitUpdater {};
+
+  passthru.scriptName = "memo.lua";
+
+  installPhase = ''
+    mkdir -p $out/share/mpv/scripts
+    cp -r memo.lua $out/share/mpv/scripts
+  '';
+
+  meta = with lib; {
+    description = "A recent files menu for mpv.";
+    homepage = "https://github.com/po5/memo";
+    license = licenses.gpl3;
```

`gpl3Only` or `gol3Plus`?

---

Sorry, I meant gpl3Only.
```suggestion
    license = licenses.gpl3Only;
```

---

fixed in [5615f1dcc5c3a6164ef5dc5d5e017f18df632e25](https://github.com/NixOS/nixpkgs/pull/288105/commits/5615f1dcc5c3a6164ef5dc5d5e017f18df632e25)